src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/model/Step.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * pottery-backend-interface - Backend API for testing programming exercises
+ * Copyright © 2018 BlueOptima Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package uk.ac.cam.cl.dtg.teaching.pottery.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.wordnik.swagger.annotations.ApiModelProperty;
+
+import java.util.Map;
+
+public class Step {
+  @ApiModelProperty("Name of this step")
+  String name;
```

Comment:
why is this package-protected?

Reply:
By mistake, will fix and document this class too.